cli/internal/helm/testdata/Azure/constellation-services/charts/ccm/templates/gcp-cm.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
I'm a little bit confused why these empty files exist but feel free to keep them if it reduces the overall complexity of the test code.

Reply:
Removed the files.